app/views/loans/viewloanaccountdetails.html

```diff
@@ -332,7 +332,7 @@ <h3>{{'label.heading.currentBalance' | translate}}: {{loandetails.currency.displ
                             </td>
                         </tr>
                         <tr>
-                            <th class="table-bold-loan">{{'label.heading.proposedamount' | translate}}</th>
+                            <th class="table-bold-loan">{{'label.heading.appliedamount' | translate}}</th>
                             <td><span class="padded-td">{{loandetails.proposedPrincipal | number}}</span></td>
```

Comment:
@tabrez-fiter  I see your still picking from proposed.

Reply:
Hi @hero78 , for this ticket it is just label change,